### ISSUE TITLE
Chapter5 error

### ISSUE THREAD

**Issue by leeyunhome** [PacktPublishing/PyTorch-Computer-Vision-Cookbook#16]
Hello,

I got some errors at Chapter5
![image](https://user-images.githubusercontent.com/59910227/108001070-d4725000-702e-11eb-9897-f2e2f8ba4db2.png)

Can you tell me how to solve this issue?

Thank you.

**Comment by mave5** [comment]
@leeyunhome  This means that `out `  is None. 
This type of error happens when the model does not find any object in the image you are feeding to the model.
Make sure that the model is trained before deploying it on an image.

**Comment by bgpantojar** [comment]
Hello,

@mravendi @leeyunhome have trained the model for 200 epoch and reaching loss function in training of 0.6 and validation of 15 and still get the same problem. Is it all right the deployment functions involved there?

![Screenshot from 2022-03-15 16-42-21](https://user-images.githubusercontent.com/35929939/158415985-39cf9dcb-1dba-4745-8c0a-aa147eb22bda.png)
![Screenshot from 2022-03-15 16-42-35](https://user-images.githubusercontent.com/35929939/158415993-937d5aed-0014-4ff1-9705-79451b6424c2.png)
